app/src/main/res/layout-sw600dp-land/onboarding_slide_final.xml

```diff
@@ -85,5 +79,12 @@
       android:layout_height="wrap_content"
       android:orientation="horizontal"
       app:layout_constraintGuide_percent="0.5" />
+
+    <androidx.constraintlayout.widget.Guideline
```

Comment:
Please don't use guidlines, as for the way we have prepared mocks, it does not guarantee the at a fix percentage the text will start or end. Avoid using guidlines in such cases and instead rely on margin/padding as they provide more robust solution and can work across devices.

Reply:
But if I'm going to remove the `guidlines`, I'll have to use some fixed widths.

Reply:
Why do we need fixed width if we remove guidelines? I mean, we have done in mobile screens without guidelines and it works fine in HDPI + Screens.

Can you please explain how will it affect the UI if we do not use guidelines and fixed width and instead simply use margins/paddings.

Reply:
I mean that if we didn't use guidelines nor fixed widths, we'll have some options here:  
1. Make a constraint between the `TextView` and the `Thumbnail` and between the `TextView` and the `Parent` then set the width of the `TextView` to `match_constraint` (0dp) and use the suitable margins
2. Make only 1 constraint between the `TextView` and the `Parent` and use the suitable margins and make the width of the `TextView` to `wrap_content`
And in both options, the text in each `TextView` will be only in one line, but in the mocks we have multiple lines.  
So, to have multiple lines like the mocks, I think we'll have to use either guidlines or fixed widths

Reply:
Okay, then in that case we will need the guidelines. Lets get the images from Chantel and then update this accordingly.